### ISSUE TITLE
Loader: crashed when NULL surface is passed to vkDestroySurfaceKHR

### ISSUE THREAD

**Issue by scyganINTEL** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1076]
According to VK+WSI spec, passing NULL to vkDestroySurfaceKHR is not invalid: 
```
If surface is not VK_NULL_HANDLE, surface must be a valid VkSurfaceKHR handle
```

However Loader crashes in terminator function: 
```
VKAPI_ATTR void VKAPI_CALL
terminator_DestroySurfaceKHR(VkInstance instance, VkSurfaceKHR surface,
                             const VkAllocationCallbacks *pAllocator) {
    struct loader_instance *ptr_instance = loader_get_instance(instance);

    VkIcdSurface *icd_surface = (VkIcdSurface *)(surface);
    if (NULL != icd_surface->real_icd_surfaces) {

```
when it tries to unpack icd_surface (which is NULL).

This affects
dEQP-vk.wsi.win32.surface.destroy_null_handle VulkanCTS test (from -1.0-dev branch)

**Comment by scyganINTEL** [comment]
This is a regression, previously it worked for me (1.0.26)

**Comment by MarkY-LunarG** [comment]
I fixed an issue like this in Master that Dominick @ AMD found on Oct 17.  Is it still happening in the tree after that fix?  That was after we shipped the 1.0.30 SDK, though.

**Comment by MarkY-LunarG** [comment]
Just verified this works for me.  It should be in next SDK drop, but, for now, use the master branch.

**Comment by MarkY-LunarG** [comment]
Commit that fixed this was:  d668bbbf9e3f1dd9e05f1064f82999fff59d059f
